hail_scripts/reference_data/config.py

```diff
@@ -89,24 +121,48 @@ def custom_mpc_select(ht):
         'select': '<Optional list of fields to select or dict of new field name to location of old field
             in the reference dataset. If '#' is at the end, we know to select the appropriate biallelic
             using the a_index.>',
-        'field_name': '<Optional name of root annotation in combined dataset, defaults to name of dataset.>',
         'custom_select': '<Optional function of custom select function>',
         'enum_select': '<Optional dictionary mapping field_name to a list of enumerated values.>'
     },
 """
 CONFIG = {
     'cadd': {
         '37': {
+            'version': 'v1.6',
             'path': 'gs://seqr-reference-data/GRCh37/CADD/CADD_snvs_and_indels.v1.6.ht',
             'select': ['PHRED'],
         },
         '38': {
+            'version': 'v1.6',
             'path': 'gs://seqr-reference-data/GRCh38/CADD/CADD_snvs_and_indels.v1.6.ht',
             'select': ['PHRED'],
         },
     },
+    'clinvar': {
+        '37': {
+            'custom_import': download_and_import_latest_clinvar_vcf,
+            'path': 'ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh37/clinvar.vcf.gz',
```

Comment:
Do you think we need a `source_file` or something in the config for storing the ingested data  paths and we could keep ht as the ingested HT? Just think all of these datasets come to us as not-HTs, either VCFs or TSVs. This keeps things special for clinvar and hgmd but potentially well want to update all references at some point.

Reply:
Yeah, that's a good point.  I've been thinking that `path` could be overloaded, but maybe making it `source_path` is more clear.  And then we can fix up the other reference datasets to align as we go forwards.

Reply:
I updated to use `source_path`.